### PR TITLE
OData Temporal Extension

### PR DESCRIPTION
**Pull request by ralfhandl**
Temporal
- [x] Starter document
- [x] Glossary
- [x] 2 Overview
  - [x] Diagrams
  - [x] Tables with example data
- [x] 3 Vocabulary
- [x] 4 Temporal Requests
  - [x] Tables with example data
  - [x] color changed cells
- [x] 5 Conformance
- [x] Appendix

Data Aggregation
- [x] aligned column formats for example data

JSON Format
- [x] references to section names in double-quotes, as in CSDL, Protocol, and in Temporal

### CODE REVIEW

File: odata-temporal-ext/3 Vocabulary.md [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+::: example
+Example ##ex: `Employees` entity set from [example model `api-1`](#api1)
+annotated with temporal terms
+```json
+"Employees": {
+  "$Collection": true,
+  "$Type": "OrgModel.Employee",
+  "@Temporal.ApplicationTimeSupport": {
+    "Timeline": {
+      "@odata.type": "https://oasis-tcs.github.io/odata-vocabularies/vocabularies/Org.OData.Temporal.V1.json#Temporal.TimelineSnapshot"
```
Comment (HeikoTheissen) [inline odata-temporal-ext/3 Vocabulary.md:53]: Break these overlong lines

File: odata-temporal-ext/4.3 Modifying Temporal Data.md [hunk 2]
```diff
@@ -0,0 +1,476 @@
...
+   specified in [OData-Protocol](#ODataProtocol).
+5. Gaps between selected time slices in the period to update are not affected.
+
+On success it returns the created or updated time slices.
+
+::: example
+Example ##ex: Change a department's budget during a period of application
+time with [`api-2`](#api2) (visible timeline)
+```json
+POST \~/Departments('D08')/history/Temporal.Update
```
Comment (HeikoTheissen) [inline odata-temporal-ext/4.3 Modifying Temporal Data.md:120]:
```suggestion
POST ~/Departments('D08')/history/Temporal.Update
```

File: odata-temporal-ext/4.3 Modifying Temporal Data.md [hunk 3]
```diff
@@ -0,0 +1,476 @@
...
+they do not significantly differ.
+:::
+
+::: example
+Example ##ex: Change an employee's job title during a period of
+application time with [`api-1`](#api1) (snapshot). Note that the period boundaries are not visible in
+[`api-1`](#api1) and are provided as `PeriodStart` and `PeriodEnd` next to the
+`Timeslice` data. The `PeriodEnd` is omitted, meaning the end of application time.
+```json
+POST \~/Employees/Temporal.Update
```
Comment (HeikoTheissen) [inline odata-temporal-ext/4.3 Modifying Temporal Data.md:222]:
```suggestion
POST ~/Employees/Temporal.Update
```

File: odata-temporal-ext/4.3 Modifying Temporal Data.md [hunk 4]
```diff
@@ -0,0 +1,476 @@
...
+   If no preceding time slice exists, the time slice is created following
+   the rules for creating entities specified in [OData-Protocol](#ODataProtocol).
+
+On success it returns the created or updated time slices.
+
+::: example
+Example ##ex: Upsert two [cost centers](#CostCenters)
+during a period of application time using the model from Example 8
+```json
+POST \~/CostCenters/Temporal.Upsert
```
Comment (HeikoTheissen) [inline odata-temporal-ext/4.3 Modifying Temporal Data.md:315]:
```suggestion
POST ~/CostCenters/Temporal.Upsert
```

File: odata-temporal-ext/3 Vocabulary.md (hunk 1 above)
Comment (ralfhandl) [reply odata-temporal-ext/3 Vocabulary.md:53]: How/why? Breaking within the string would make it invalid JSON, and just moving the overlong string into a separate line wouldn't help much.

Comment (HeikoTheissen) [reply odata-temporal-ext/3 Vocabulary.md:53]:
```suggestion
      "@odata.type": "https://oasis-tcs.github.io/odata-vocabularies/
        vocabularies/Org.OData.Temporal.V1.json#Temporal.TimelineSnapshot"
```
looks better than

image

Comment (ralfhandl) [reply odata-temporal-ext/3 Vocabulary.md:53]: I feel uncomfortable writing down incorrect JSON just for better looks.

**Review by HeikoTheissen — APPROVED** [review]

**Pull request merged**